assets/app/views/createRoute.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <ol class="breadcrumb">
+        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+        <li class="active"><strong>Add Route</strong></li>
+      </ol>
+      <alerts alerts="alerts"></alerts>
+      <h1>Add Route</h1>
+      <div ng-if="serviceName">
+        <span class="help-block">Create a route for service {{serviceName}}.</span>
+      </div>
+      <div ng-if="!serviceName">
+        <span class="help-block">Select a service and enter route details.</span>
```

Comment:
not that you shouldnt be able to, just mean users that dont know what they are doing wont know why they need to select a service